app/assets/javascripts/services/event_notifications_service.js

```diff
@@ -299,9 +299,13 @@ angular.module('ManageIQ').service('eventNotifications', ['$timeout', '$window',
     }
   };
 
-  this.showToast = function(notification, persist) {
+  var simpleShow = function(notification, persist) {
     var $this = this;
```

Comment:
neither `simpleShow()` nor `_.throttle(SimpleShow)()` will give the right `this`, needs to be outside..